assets/src/containers/AssignmentPlanning.js

```diff
@@ -37,22 +37,35 @@ const styles = theme => ({
   }
 })
 
+const getCurrentWeek = assignmentData => {
+  // default week
+  let currentWeek = 0
```

Comment:
@pushyamig if you want to test, change `currentWeek` here to be, say , `5`, and you'll see the table will scroll to the 6th element (since index starts at 0).